File: Flight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApiPart.Models
{
    public class Flight
    {
        public int Id { get; set; }
        public string AirlineCompanyId { get; set; }
        public string OriginCountryCode { get; set; }
        public string DestinationCountryCode { get; set; }
        public DateTime DepartureTime { get; set; }
        public DateTime LandingTime { get; set; }
        public int RemainingTickets { get; set; }
        public Flight()
        {

        }
        public Flight(int id, string airlineCompanyId, string originCountryCode, string destinationCountryCode, DateTime departureTime, DateTime landingTime, int remainingTickets)
        {
            Id = id;
            AirlineCompanyId = airlineCompanyId;
            OriginCountryCode = originCountryCode;
            DestinationCountryCode = destinationCountryCode;
            DepartureTime = departureTime;
            LandingTime = landingTime;
            RemainingTickets = remainingTickets;
        }
        public override string ToString()
        {
            return $"Id {Id} AirlineCompanyId {AirlineCompanyId} OriginCountryCode {OriginCountryCode} " +
                $"DestinationCountryCode {DestinationCountryCode} DepartureTime {DepartureTime} " +
                $"LandingTime {LandingTime} RemainingTickets {RemainingTickets}";
        }
    }
}

File: flight.js
class Flight {
    constructor(id, airlineCompanyId, originCountryCode, destinationCountryCode, departureTime, landingTime, remainingTickets, status) {
        this.Id = id
        this.AirlineCompanyId = airlineCompanyId
        this.OriginCountryCode = originCountryCode
        this.DestinationCountryCode = destinationCountryCode
        this.DepartureTime = departureTime
        this.LandingTime = landingTime
        this.RemainingTickets = remainingTickets
        if (status == undefined) {
            this.status = "Landing..."
        }

    }
    
}
class David {
    name;
    age;


}